frontend/src/components/authentication/ForgotPassword.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {useState, useCallback} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+import {CssBaseline, TextField} from '@material-ui/core'
+import {Button, Avatar, Typography, Link, Paper, Grid, Box }  from '@material-ui/core'
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined'
+
+import * as gridSize from '../../constants/styles.values'
+import {SIX as SHADOW_DEPTH_SIX} from '../../constants/styles.values'
+import { recover } from '../../actions/authActions'
+import { RootState } from '../../reducers/index'
+import {useStyles} from './SignUp.styles'
+
+export default function ForgotPasswordSide(): React.ReactElement {
+
+  const custom = useSelector((state: RootState) => state.authReducer)
+
+    const [email, setEmail] = useState('')
+    const onChangeEmail = useCallback((e) => {
+      setEmail(e.target.value)
+    }, [setEmail])
```

Comment:
What should I change here?

Reply:
Ok, only it will be 
```
const onChangeEmail = useCallback((e) => {
      setEmail(e.target.value)
}, [])
```
Because the function useCallback takes 2 arguments